fix(ktable): doc revalidate usage [khcp-4670] (#805)
Kong/kongponents: docs/components/table.md

</script>
```

> An alternative usage is with the native `key` attribute in conjuction with a `fetcher` and SWRV `revalidate` function. In this case you will want the `key` to start at `0`.

Reviewer: ```suggestion
An alternate implementation is to apply a `key` attribute to the `KTable` in conjunction with a `fetcher` and the SWRV `revalidate` function. To prevent unnecessary calls on mount, the `key` `ref` should have an initial value of `0`.
```

> Since the `fetcher` function will handle the intial GET of the data, we want the cache key for the `revalidate` function to evaluate to `falsey` on page load (to avoid an unnecessary duplicate call), and will manually call `revalidate()` and increment the `key` to trigger a refetch and redraw of the table.

Reviewer: nit: Add an empty line between the paragraphs
```suggestion

Since the `fetcher` function will handle the intial GET of the data, we want the cache key for the `revalidate` function to evaluate to `falsey` on page load (to avoid an unnecessary duplicate call), and will manually call `revalidate()` and increment the `key` to trigger a refetch and redraw of the table.
```

```html
<template>
  <KTable :key="key" :fetcher="fetcher" :headers="headers" />
</template>

<script lang="ts">
import { defineComponent } from 'vue'

export default defineComponent({
  setup () {
    const key = ref(0) // initialized to zero
    const fetcher = async ({ pageSize, page, query, offset = null }) => {
      try {
        const res = await sniServices.getAll(pageSize, offset)

        // handle data
      } catch (error) {
        // handle error
      }
    }

    const { revalidate } = composables.useRequest(

>       () => key.value && `snis-list-${key.value}`, // will eval falsey on load, which is what we want

Reviewer: ```suggestion
      () => key.value && `snis-list-${key.value}`, // will evaluate to falsey on mount, preventing an extra call
```

</script>
```

An alternate implementation is to apply a `key` attribute to the `KTable` in conjunction with a `fetcher` and the SWRV `revalidate` function. To prevent unnecessary calls on mount, the `key` `ref` should have an initial value of `0`.

Since the `fetcher` function will handle the intial GET of the data, we want the cache key for the `revalidate` function to evaluate to `falsey` on page load (to avoid an unnecessary duplicate call), and will manually call `revalidate()` and increment the `key` to trigger a refetch and redraw of the table.

```html
<template>
  <KTable :key="key" :fetcher="fetcher" :headers="headers" />
</template>

<script lang="ts">
import { defineComponent } from 'vue'

export default defineComponent({
  setup () {
    const key = ref(0) // initialized to zero
    const fetcher = async ({ pageSize, page, query, offset = null }) => {
      try {

>         const res = await sniServices.getAll(pageSize, offset)

Reviewer: ```suggestion
        const res = await services.getAll(pageSize, offset)
```

        // handle data
      } catch (error) {
        // handle error
      }
    }

    const { revalidate } = composables.useRequest(

>       () => key.value && `snis-list-${key.value}`, // will evaluate to falsey on mount, preventing an extra call

Reviewer: ```suggestion
      () => key.value && `service-list-${key.value}`, // will evaluate to falsey on mount, preventing an extra call
```

      () => { return fetcher() }
    )

    const handleDelete = (id) => {
      try {

>         const res = await sniServices.delete(id)

Author: ```suggestion
        const res = await services.delete(id)
```
